openmls/src/ciphersuite/hpke.rs

```diff
@@ -71,17 +79,34 @@ impl From<CryptoError> for Error {
 /// Context for HPKE encryption
 #[derive(Debug, Clone, TlsSerialize, TlsDeserialize, TlsDeserializeBytes, TlsSize)]
 pub struct EncryptContext {
+    /// Prefixed with LABEL_PREFIX
     label: VLBytes,
     context: VLBytes,
 }
 
 impl EncryptContext {
     /// Create a new [`EncryptContext`] from a string label and the content bytes.
-    pub fn new(label: &str, context: VLBytes) -> Self {
+    /// Ensures that the prefix LABEL_PREFIX is prepended to the label.
+    pub(crate) fn new(label: &str, context: VLBytes) -> Self {
         let label_string = LABEL_PREFIX.to_owned() + label;
         let label = label_string.as_bytes().into();
         Self { label, context }
     }
```

Comment:
Without newlines the code gets really hard to read.

```suggestion
    }

```